components/org.wso2.analytics.apim.widgets/APIMApiLatencyTime/src/APIMApiLatencyWidget.jsx

```diff
@@ -396,16 +396,17 @@ class APIMApiLatencyWidget extends Widget {
         this.resetState();
         const queryParam = super.getGlobalState(queryParamKey);
         const { apiCreatedBy } = queryParam;
-        const { providerConfig, apiDataList, username } = this.state;
+        const { providerConfig, username } = this.state;
         const { id, widgetID: widgetName } = this.props;
 
+        let { apiDataList } = this.state;
```

Comment:
use object destructuring default values. so that you can remove `apiDataList && apiDataList.length > 0` check
```suggestion
        let { apiDataList=[]} = this.state;
```